<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Restaurant Software</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/favicon.png</normaloff>../icon/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QStackedWidget" name="uimanager">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="dashboardpage"/>
      <widget class="QWidget" name="itemspage">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_name">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="items_name_input">
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>35</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_category">
                <property name="text">
                 <string>Category</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="items_category_combo">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
                <property name="maxVisibleItems">
                 <number>20</number>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAtCurrent</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lable_price">
                <property name="text">
                 <string>Price</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="items_price_input">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_description">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="items_description_input">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_vn">
              <property name="text">
               <string>Meal Type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="items_mtype_combo">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Veg</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Non-Veg</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_items_btn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: #FFFFFF;
	color: #666666;
}</string>
              </property>
              <property name="text">
               <string>ADD</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="3,2">
            <item>
             <widget class="QListWidget" name="items_list_widget">
              <property name="contextMenuPolicy">
               <enum>Qt::DefaultContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QListWidget {
	background-color: #FFFFFF;
	border: 1px solid #ccc;
	font-size: 18px;
	font-weight: bold;
}

QListWidget::item {
	width: 150px;
	height: 150px;
	margin: 5px;
	color: #666666;
}</string>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="autoScrollMargin">
               <number>20</number>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::IgnoreAction</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="textElideMode">
               <enum>Qt::ElideMiddle</enum>
              </property>
              <property name="movement">
               <enum>QListView::Static</enum>
              </property>
              <property name="flow">
               <enum>QListView::LeftToRight</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>true</bool>
              </property>
              <property name="layoutMode">
               <enum>QListView::SinglePass</enum>
              </property>
              <property name="gridSize">
               <size>
                <width>130</width>
                <height>150</height>
               </size>
              </property>
              <property name="viewMode">
               <enum>QListView::IconMode</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="items_table_widget">
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="categorypage">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,10,1">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Category Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="category_name_input">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	padding: 0 10px
}</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="cursorPosition">
               <number>0</number>
              </property>
              <property name="placeholderText">
               <string>Enter Category Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_category_btn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: #FFFFFF;
	color: #666666;
}</string>
              </property>
              <property name="text">
               <string>ADD</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="8,2">
            <item>
             <widget class="QListWidget" name="category_list_widget">
              <property name="styleSheet">
               <string notr="true">QListWidget {
	background-color: #ffffff;
	border: 1px solid #ccc;
	font-size: 15px;
	font-weight: bold;
}

QListWidget::item {
	width: 100px;
	height: 100px;
	white-space: no-wrap;
	margin: 5px;
	color: #666666;
	padding: 10px;
	background: #E1F8DC;
	border-radius: 10px;

}</string>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::IgnoreAction</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="movement">
               <enum>QListView::Static</enum>
              </property>
              <property name="viewMode">
               <enum>QListView::IconMode</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="category_table_widget"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tablespage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,10,1">
            <item>
             <widget class="QLabel" name="table_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Table Name / No.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="table_name_input">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	padding: 0 10px
}</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="cursorPosition">
               <number>0</number>
              </property>
              <property name="placeholderText">
               <string>Enter Table Name or Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_table_btn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: #FFFFFF;
	color: #666666;
}</string>
              </property>
              <property name="text">
               <string>ADD</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="8,2">
            <item>
             <widget class="QListWidget" name="table_list_widget">
              <property name="styleSheet">
               <string notr="true">QListWidget {
	background-color: #ffffff;
	border: 1px solid #ccc;
	font-size: 15px;
	font-weight: bold;
}

QListWidget::item {
	width: 100px;
	height: 100px;
	white-space: no-wrap;
	margin: 5px;
	color: #666666;
	padding: 10px;
	background: #E1F8DC;
	border-radius: 10px;

}</string>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::IgnoreAction</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="movement">
               <enum>QListView::Static</enum>
              </property>
              <property name="viewMode">
               <enum>QListView::IconMode</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="table_table_widget"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Master</string>
    </property>
    <addaction name="actionDashboard"/>
    <addaction name="actionItems"/>
    <addaction name="actionCategory"/>
    <addaction name="actionTables"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionDatabase_Settings"/>
    <addaction name="actionCurrency"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDeveloper"/>
   </widget>
   <widget class="QMenu" name="menuInvoices">
    <property name="title">
     <string>Orders</string>
    </property>
    <addaction name="actionOrders"/>
    <addaction name="actionReports"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuInvoices"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionItems">
   <property name="text">
    <string>Items</string>
   </property>
  </action>
  <action name="actionCategory">
   <property name="text">
    <string>Category</string>
   </property>
  </action>
  <action name="actionOrders">
   <property name="text">
    <string>Orders</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>Reports</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="text">
    <string>Developer</string>
   </property>
  </action>
  <action name="actionDatabase_Settings">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionCurrency">
   <property name="text">
    <string>Currency</string>
   </property>
  </action>
  <action name="actionDashboard">
   <property name="text">
    <string>Dashboard</string>
   </property>
  </action>
  <action name="actionTables">
   <property name="text">
    <string>Tables</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
